packages/cpp/README.md

```diff
@@ -81,14 +81,14 @@ To get this working, you need to enable clangd's global index using the
 
 ## Using the clang-tidy linter
 
-Note: This functionality is available when using clangd 9 and later.
+**Note: This functionality is available when using clangd 9 and later.**
 
-You can set the preference 'cpp.clangTidy' to enable the clang-tidy linter included in clangd. When the preference is set, there are two ways to chose which of its built-in checks clang-tidy will use:
+You can set the preference 'cpp.clangTidy' to enable the clang-tidy linter included in clangd. When the preference is set, there are two ways to choose which of its built-in checks clang-tidy will use:
```

Comment:
Instead of **set** in `...when the preference is set...`, shouldn't it be **enabled**?
Set implies it has a value but in this case we want to state that when the preference is enabled you can use the following configuration options.

Reply:
Done